File: 06-operaciones-aritmeticas/php/aritmetica.php
<?php

$a = floatval(readline("Introduzca un número: "));
$b = floatval(readline("Introduzca otro número: "));
$operacion = readline("Introduzca la operación a realizar (+, -, *, /): ");

if ($operacion == "+") {
    echo "El resultado de la suma es: " . ($a + $b);
} elseif ($operacion == "-") {
    echo "El resultado de la resta es: " . ($a - $b);
} elseif ($operacion == "*") {
    echo "El resultado de la multiplicación es: " . ($a * $b);
} elseif ($operacion == "/") {
    echo "El resultado de la división es: " . ($a / $b);
} else {
    echo "Operación no válida";
}

File: 06-operaciones-aritmeticas/python/aritmetica.py
a = float(input("Introduzca un número: "))
b = float(input("Introduzca otro número: "))
operacion = input("Introduzca la operación a realizar (+, -, *, /): ")

if operacion == "+":
    resultado = a + b
elif operacion == "-":
    resultado = a - b
elif operacion == "*":
    resultado = a * b
elif operacion == "/":
    resultado = a / b
else:
    print("Operación no válida")
    exit()

print(f"El resultado de la operación es: {resultado:.2f}")
